### DISCUSSION TITLE
After executing the script I'm getting error

### DISCUSSION THREAD

**Discussion by Kurra432** [learndevopsonline/learndevopsonline#1937] [Q&A]
After cloning the repository , executing the script getting segmentation error 
![Segmentation error](https://github.com/user-attachments/assets/483ac8cc-30d0-4bdc-8982-41709b0112d2)

**Comment by chaithanya1812** [comment]
Hi @Kurra432 , Watch video one more time, It wil be oky
![image](https://github.com/user-attachments/assets/86018e9d-fbed-47f8-b7ae-3535ebc997ee)

**Comment by Kurra432** [comment]
It's working now
